internal/cli/roles_permissions.go

```diff
@@ -149,9 +147,9 @@ func addRolePermissionsCmd(cli *cli) *cobra.Command {
 			}
 
 			var rs *management.ResourceServer
-			rs, err := cli.api.ResourceServer.Read(cmd.Context(), url.PathEscape(inputs.APIIdentifier))
```

Comment:
Same as above.

Reply:
This is already done within the Go SDK: https://github.com/auth0/go-auth0/blob/a3d8aa98408d89c10f9be24002cf217c5f2f7e32/management/management_request.go#L29